src/classes/ALLO_Allocations_TDTM.cls

```diff
@@ -266,11 +282,18 @@ public class ALLO_Allocations_TDTM extends TDTM_Runnable {
         for (integer i=0; i<triggerList.size(); i++) {  
             Allocation__c allo = triggerlist[i];
 
+            //throw error if allocating at the payment level but default allocation is not enabled
+            if (!settings.Default_Allocations_Enabled__c && allo.Payment__c != null) {
```

Comment:
Looks like we need to expand this error to manually created Payment Allocations as well? Using a related list, I've been able to create Payment Allocations even with Default Allocations set to false.